### PR TITLE
PMM-3476 Sync constraints with models.

### PR DESCRIPTION
**Pull request by DexterHD**

### CODE REVIEW

File: models/database.go [hunk 1]
```diff
@@ -65,7 +65,17 @@ var databaseSchema = [][]string{
 			UNIQUE (node_name),
 			UNIQUE (machine_id),
 			UNIQUE (docker_container_id),
-			UNIQUE (address, region)
+			UNIQUE (address, region),
+
+			CHECK (node_type <> ''),
```
Comment (AlekSi) [inline models/database.go]: Can we make them inline? It is not different from `NOT NULL` for our purposes.

File: models/database.go [hunk 2]
```diff
@@ -40,26 +40,26 @@ var databaseSchema = [][]string{
 
 		`CREATE TABLE nodes (
 			-- common
-			node_id VARCHAR NOT NULL,
-			node_type VARCHAR NOT NULL,
-			node_name VARCHAR NOT NULL,
-			machine_id VARCHAR,
+			node_id VARCHAR NOT NULL ,
```
Comment (AlekSi) [inline models/database.go]:
```suggestion
			node_id VARCHAR NOT NULL,
```

**Comment by AlekSi** [comment]
Waiting for CI…

**Comment by DexterHD** [comment]
@AlekSi Can be merged I think, but as you an Assignee I can't do it. :)

**Pull request merged**